core/include/traccc/seeding/detail/singlet.hpp

```diff
@@ -30,32 +30,10 @@ inline TRACCC_HOST_DEVICE bool operator!=(const sp_location& lhs,
     return (lhs.bin_idx != rhs.bin_idx || lhs.sp_idx != rhs.sp_idx);
 }
 
-/// Container of singlet belonging to one detector module
-template <template <typename> class vector_t>
-using singlet_collection = vector_t<sp_location>;
+/// Declare all singlet collection types
+using singlet_collection_types = collection_types<sp_location>;
 
-/// Convenience declaration for the singlet collection type to use in host code
-using host_singlet_collection = singlet_collection<vecmem::vector>;
+/// Declare all singlet container types
+using singlet_container_types = container_types<unsigned int, sp_location>;
```

Comment:
Yeah, let's just drop these definitions. If they are not needed, let's not burden the compiler with them.